### PR TITLE
EZP-30750: Added possibility to set purge_type with environmental variables

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZP-30750

### CODE REVIEW

**Comment by adamwojs** [comment]
CI failure seems to be releated to PR changes.

**Comment by webhdx** [comment]
CI tests using Varnish & Redis seem to be fine: https://github.com/ezsystems/ezplatform-page-builder/pull/453

File: config/packages/ezplatform.yaml [hunk 1]
```diff
@@ -1,19 +1,16 @@
 imports:
+    - { resource: env/generic.php }
+    - { resource: env/platformsh.php }
+    - { resource: cache_pool/cache.memcached.yaml }
```
Comment (emodric) [inline config/packages/ezplatform.yaml]: Why uncommenting of memcached and redis pool services?

Comment (webhdx) [reply config/packages/ezplatform.yaml]: Yes it shouldn't be done this way but I didn't know that `env/generic.php` was loading those files based on `CACHE_POOL` value. It wasn't working tho so I fixed it and removed those imports completely.

**Comment by webhdx** [comment]
Ping @mnocon. Ready for another round of testing. I think we should also test if p.sh instances are configured correctly as I had to move/fix some of its configuration.

**Comment by lserwatka** [comment]
Well done @webhdx 🥇

**Comment by lserwatka** [comment]
@webhdx could you merge it horizontally?

**Comment by webhdx** [comment]
Merged

**Pull request merged**